core/types.rs

```diff
@@ -40,7 +40,7 @@ impl Display for OwnedValue {
         match self {
             OwnedValue::Null => write!(f, "NULL"),
             OwnedValue::Integer(i) => write!(f, "{}", i),
-            OwnedValue::Float(fl) => write!(f, "{}", fl),
+            OwnedValue::Float(fl) => write!(f, "{:?}", fl),
```

Comment:
Modified this since the float value trailing zero was getting truncated (ex: 79.0 was printed as 79)